### DISCUSSION TITLE
Articles about Trust

### DISCUSSION THREAD

**Discussion by trbouma** [trustoverip/tswg-trust-registry-tf#76] [General]
From the Toronto Star - Tech treats trust as a weakness to be exploited. 

https://pressreader.com/article/281706913863942

**Comment by mathieuglaude** [comment]
- I think in the crypto example, he confuses trust in an "unalterable glorified spreadsheet" rather than in incentives. An append-only model on its own doesn't help build trust around its data. Incentives are built into the protocol. Unsure why the FTX example is relevant at all.
- The ChatGPT one is interesting. Trust won't come from the LLM, but from the 'trust layer' built on top of them. The general LLMs will basically become commoditized over time. The way the layer above it synthesizes it to present it to the user in the way they intend to is where I think the trust will be built/lost. This is a layer btw that I think should be transparent/open so users know why they're seeing what they are. Social media algorithms will need to follow the same path I'd think.
